railties/test/generators/app_generator_test.rb

```diff
@@ -117,35 +116,33 @@ def test_application_name_is_detected_if_it_exists_and_app_folder_renamed
 
     run_generator [app_root]
 
-    Rails.application.config.root = app_moved_root
-    Rails.application.class.stubs(:name).returns("Myapp")
-    Rails.application.stubs(:is_a?).returns(Rails::Application)
+    stub_rails_application(app_moved_root) do
+      Rails.application.stub(:is_a?, proc{ Rails::Application }) do
```

Comment:
Without `proc` `Rails::Application` gives error "Rails::Application is abstract, you cannot instantiate it directly." I'm not sure if it's right way to do.
Also other tests are running properly without stubbing `is_a?` so removed it from other tests.